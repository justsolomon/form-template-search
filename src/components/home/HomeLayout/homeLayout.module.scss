@import 'styles/variables';

.home {
  width: 80%;
  margin: 80px auto;

  @media #{$media-desktop} {
    width: 90%;
  }

  @media #{$media-mobile--sm} {
    margin: 40px auto;
  }
}
