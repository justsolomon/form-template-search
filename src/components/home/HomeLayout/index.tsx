import styles from './homeLayout.module.scss';

interface HomeLayoutProps {
  children: React.ReactNode;
}

const HomeLayout = ({ children }: HomeLayoutProps) => {
  return <div className={styles['home']}>{children}</div>;
};

export default HomeLayout;
