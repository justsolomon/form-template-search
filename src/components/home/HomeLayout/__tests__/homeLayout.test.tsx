import { render } from '@testing-library/react';
import HomeLayout from '..';

describe('<HomeLayout />', () => {
  it('should render the HomeLayout component correctly', () => {
    const { asFragment } = render(<HomeLayout>Test children</HomeLayout>);

    expect(asFragment).toMatchSnapshot();
  });
});
