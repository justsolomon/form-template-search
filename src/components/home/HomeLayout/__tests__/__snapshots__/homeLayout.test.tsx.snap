// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeLayout /> should render the HomeLayout component correctly 1`] = `[Function]`;
